SpywareFilter/sections/specific.txt

```diff
@@ -25,6 +25,8 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com
 !
+||stuttgarter-zeitung.de/*/tracking/tracking.js
```

Comment:
It is blocked 
![image](https://user-images.githubusercontent.com/8361299/113584304-68c26180-9633-11eb-86b4-aee1a5f822c1.png)

Reply:
Removed